### ISSUE TITLE
maxcdn.bootstrapcdn.com provides expires certs

### ISSUE THREAD

**Issue by gemal** [MaxCDN/bootstrapcdn#1567]
https://www.ssllabs.com/ssltest/analyze.html?d=maxcdn.bootstrapcdn.com&s=2001%3a4de0%3aac19%3a0%3a0%3a1%3ab%3a3b&latest

![image](https://user-images.githubusercontent.com/69105/83384359-683fb580-a3e7-11ea-8c2c-3342bbcfc421.png)

![image](https://user-images.githubusercontent.com/69105/83384365-6aa20f80-a3e7-11ea-81c7-4a94d8e84209.png)

**Comment by XhmikosR** [comment]
Thanks for the issue and sorry for this issue. I deferred the issue to StackPath's support team, and it should be fixed soon.

**Comment by jdorfman** [comment]
Working with StackPath, I got no alert, this was automated, trying to figure out why it didn't auto-renew. Unacceptable.

**Comment by jdorfman** [comment]
Ok after some investigating the issue was with the root certificate provider (Comodo). The issue has been fixed. We are waiting on a post-mortem from StackPath to address future steps on how to avoid this in the future.

Don't close this issue until then. Thanks.
